builder/builder.go

```diff
@@ -106,6 +106,9 @@ func (b *Builder) onSealedBlock(block *types.Block, ordersClosedAt time.Time, se
 
 	value := new(boostTypes.U256Str)
 	err = value.FromBig(block.Profit)
+
+	log.Info("Block profit for block", value)
```

Comment:
It has to be  `log.Info("message", "value name", value)` otherwise it will print errors.